tests/bsim/bluetooth/host/privacy/peripheral/src/dut_rpa_expired.c

```diff
@@ -162,5 +169,5 @@ void dut_rpa_expired_procedure(void)
 	start_rpa_advertising();
 
 	/* Nothing to do */
-	PASS("PASS\n");
+	TEST_PASS("PASS");
```

Comment:
Not now, but maybe later `TEST_PASS` should be refactored to print "Pass" automatically and perhaps with the test_id as well